contrib/debian/netdata-plugin-apps.postinst

```diff
@@ -5,6 +5,7 @@ set -e
 case "$1" in
   configure|reconfigure)
     setcap "cap_dac_read_search=eip cap_sys_ptrace=eip" /usr/libexec/netdata/plugins.d/apps.plugin
+    chown root:netdata /usr/libexec/netdata/plugins.d/apps.plugin
```

Comment:
Need to be swapped. Apparently `chown` resets caps

Reply:
Good catch, I completely forgot about that.